botfront/imports/lib/utils.js

```diff
@@ -88,6 +89,8 @@ export const getProjectIdFromModelId = modelId => Projects.findOne({ nlu_models:
 
 export const getModelIdsFromProjectId = projectId => Projects.findOne({ _id: projectId }, { fields: { nlu_models: 1 } }).nlu_models;
 
+export const getLanguagesFromProjectId = projectId => getNluModelLanguages(getModelIdsFromProjectId(projectId));
+
```

Comment:
Are those change still necessary? I thought that all the logic was now handled in Rasa